src/routes/indexRouter.js

```diff
@@ -1,9 +1,9 @@
-const productsRouter = require("./productsRouter");
+// const productsRouter = require("./productsRouter");
```

Comment:
Remove comment line

Reply:
there was an error about the commented line, we can't erase it, just uncomment it.